src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/InternalCaseCreationContactDetailsAppender.java

```diff
@@ -19,19 +19,19 @@
 public class InternalCaseCreationContactDetailsAppender implements PreSubmitCallbackHandler<AsylumCase> {
 
```

Comment:
There is no changes here, just some alignment. Can you please rollback this file locally and then push this commit?